### PR TITLE
 updated my profile with milestone

### PR DESCRIPTION
**Pull request by kunal00000**
## Changes proposed

Added Milestones to my profile.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7641"></a>

### CODE REVIEW

File: data/kunal00000.json [hunk 1]
```diff
@@ -46,5 +37,30 @@
...
       "icon": "FaInstagram"
     }
+  ],
+  "milestones": [
+    {
+      "title": "Postman Student Expert",
+      "date": "May 2023",
+      "icon": "SiPostman",
+      "color": "orange",
+      "description": "Became a Postman Student Expert after completing the API Fundamentals",
```
Comment (Pradumnasaraf) [inline data/kunal00000.json:47]:
```suggestion
      "description": "Became a Postman Student Expert after completing the API Fundamentals"
```

**Review by Pradumnasaraf — DISMISSED** [review]
Hey,

There is an extra `,`. Please remove that

File: data/kunal00000.json (hunk 1 above)
Comment (kunal00000) [reply data/kunal00000.json]: Hi, 
Thanks for pointing out Pradumna, commited the changes.
Btw how did you write this suggestion, can you share text of this 🙂.

File: data/kunal00000.json [hunk 2]
```diff
@@ -1,24 +1,15 @@
 {
   "name": "Kunal Verma",
+  "type": "personal",
+  "displayStatsPublic": true,
```
Comment (SaraJaoude) [inline data/kunal00000.json:4]: this is no longer required:
```suggestion
```

File: data/kunal00000.json [hunk 3]
```diff
@@ -1,24 +1,13 @@
 {
   "name": "Kunal Verma",
   "bio": "**Open Source** | Frontend WebDeveloper",
-  "tags": ["Open Source", "MERN", "Tailwind", "Next Js", "MySQL"],
+  "tags": ["Open Source", "Typescript", "MERN", "Tailwind", "Next Js", "MySQL"],
+  "avatar": "https://github.com/kunal00000.png",
```
Comment (SaraJaoude) [inline data/kunal00000.json:5]: this is also no longer necessary:
```suggestion
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**